prediction_market_agent/agents/microchain_agent/deploy.py

```diff
@@ -51,6 +51,19 @@ def run(
         prompt_handler.save_prompt(editable_prompt)
 
 
-class DeployableMicrochainModifiableSystemPromptAgent(DeployableMicrochainAgent):
+class DeployableMicrochainModifiableSystemPromptAgent0(DeployableMicrochainAgent):
     system_prompt_choice: SystemPromptChoice = SystemPromptChoice.JUST_BORN
     load_historical_prompt: bool = True
+    task_description = AgentIdentifier.MICROCHAIN_AGENT_OMEN_LEARNING_0
```

Comment:
This could be set via env var, and then you wouldn't need to duplicate this python code, and all the duplication would just be in the terraform modules

Reply:
I know but I actually liked that we have this explicit list of available agents: https://github.com/gnosis/prediction-market-agent/blob/8898be69854172abae0552ed2c5d3d037bbdeb53/prediction_market_agent/run_agent.py#L48

But I agree there is a ton of duplication, what do you think about it now? I did a small refactor.

Reply:
I'm not sure. My instinct is that multiple copies of the same agent should be specified only in the terraform code. But by that logic I should have made `load_historical_prompt` settable by env var too, and not have `DeployableMicrochainModifiableSystemPromptAgent` either. So happy to go with this for now :)